### PR TITLE
[EAE-101] When a new user registers, they can select multiple areas of interest on registration form

### PR DESCRIPTION
**Pull request by glaubermagal**

### CODE REVIEW

**Review by fabian-eae — COMMENTED** [review]
I do not see the need for choices.js this could be done with:
- bootstrap icons
- vanilla JS
- vanilla CSS

File: assets/javascripts/subscribe.js [hunk 1]
```diff
@@ -86,24 +85,27 @@ import modal from "/lib/modal.js";
...
+			} else {
+				const option = document.createElement('option');
+				option.disabled = true;
+				option.selected = true;
+				option.innerText = f.h;
+				i.append(option);
 			}
 
+			f.o.forEach(optionValue => {
+				const optionElement = document.createElement('option');
```
Comment (fabian-eae) [inline assets/javascripts/subscribe.js:101]: our variable naming convention is:

- no camelcase
- small single letters for variables used in iterations (optional)

**Comment by glaubermagal** [comment]
> I do not see the need for choices.js this could be done with:
> 
> * bootstrap icons
> * vanilla JS
> * vanilla CSS
 
Why not use a solution that already exists like https://github.com/Choices-js/Choices? It would be so much complex to build the same solution from scratch

File: assets/javascripts/subscribe.js (hunk 1 above)
Comment (glaubermagal) [reply assets/javascripts/subscribe.js]: ok, I believe we should create a documentation later explaining the specific good practices used in this project

**Comment by fabian-eae** [comment]
> 
> Why not use a solution that already exists like https://github.com/Choices-js/Choices? It would be so much complex to build the same solution from scratch

don't worry about it. it wasn't complex :)

**Pull request closed without merging**